src/Frontend/Core.hs

```diff
@@ -12,6 +13,61 @@ import Lucid
 import Lucid.Base
 import Network.Wai.Internal (Response(ResponseFile, ResponseBuilder, ResponseStream, ResponseRaw))
 import Network.Wai (Middleware)
+import Text.Digestive.View
+
+----------------------------------------------------------------------
+-- building blocks
```

Comment:
having this in Frontend.Core is fine.  alternatives:

1. leave in Frontend.Html
1. move to Frontend.PageCore
1. move to Frontend.Page.Core

i'm ok with all, but don't like the fact that Frontend.Page.Core makes it look like it's just another page.